test/RecipeRepository-test.js

```diff
@@ -1,8 +1,52 @@
 import { expect } from 'chai';
 import RecipeRepository from '../src/classes/RecipeRepository';
+import { recipeData } from '../src/data/recipes';
+import { ingredientsData } from '../src/data/ingredients'
 
 describe('Recipe', () => {
+  let repo;
+
+  beforeEach(() => {
+    repo = new RecipeRepository(recipeData);
+  });
+
   it('Should be a function', () => {
     expect(RecipeRepository).to.be.a('function');
   });
-})
\ No newline at end of file
+  
+  it('Should be an instance of Recipe', () => {
+    expect(repo).to.be.an.instanceOf(RecipeRepository);
+  });
+
+  it('Should have all recipes as properties', () => {
+    expect(repo[0].id).to.equal(595736);
```

Comment:
Since repo is an object, would it need to have a this.recipes property that is assigned to an array of recipe objects? so to access the first recipe, repo.recipes[0].id to equal 595736.